ansible/plugins/inventory/nodejs_yaml.py

```diff
@@ -45,10 +45,10 @@
     'type': ('infra', 'release', 'test'),
 
     # providers - validated for consistency
-    'provider': ('azure', 'digitalocean', 'joyent', 'ibm', 'linuxonecc',
+    'provider': ('azure', 'digitalocean', 'joyent', 'linuxonecc',
                  'macstadium', 'marist', 'mininodes', 'msft', 'osuosl',
                  'rackspace', 'requireio', 'scaleway', 'softlayer', 'voxer',
-                 'packetnet', 'nearform')
+                 'packetnet', 'pcc', 'nearform')
```

Comment:
typo? Dawson suggested `use "pdp" to stand for "Power Development Cloud"`

Reply:
on the call we had with monday they said they got this order from pdp - they are actually `Poughkeepsie Client Center" I believe

Reply:
Internal conversation ended with "lets just call this `ibm`".